wire-library/wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/RealGrpcStreamingCall.kt

```diff
@@ -61,6 +65,11 @@ internal class RealGrpcStreamingCall<S : Any, R : Any>(
       }
     }
 
+    scope.launch(Dispatchers.IO) {
+      requestChannel.writeToRequestBody(requestBody, grpcMethod.requestAdapter, call)
+    }
+    call.enqueue(responseChannel.readFromResponseBodyCallback(grpcMethod.responseAdapter))
```

Comment:
Hmm we should probably attach this to the scope and cancel the call when the scope is canceled. It should be something like
```kotlin
scope.coroutineContext[Job]!!.invokeOnCancelation {
  call.cancel()
}
```

Reply:
Hmm. . . . I’m gonna save that for a follow-up. I want to test it properly!